.changeset/eleven-mangos-mix.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-css': minor
+'@itwin/itwinui-react': minor
+---
+
+Add small size toggle switch option using `size` prop as follow: `<ToggleSwitch size='small' />`
```

Comment:
```suggestion
Add small size toggle switch option using `size` prop as follows: `<ToggleSwitch size='small' />`
```

Reply:
Hold on, changing variable name to `size` was breaking my code which was why I named it `switchSize`. However it is not anymore but it is breaking the unit test. Do you happen to know why?

`
  ● Test suite failed to run

    src/core/ToggleSwitch/ToggleSwitch.test.tsx:35:46 - error TS2322: Type 'string' is not assignable to type 'undefined'.

    35   const { container } = render(<ToggleSwitch size='small' />);
                                                    ~~~~

      src/core/ToggleSwitch/ToggleSwitch.tsx:33:3
        33   size?: 'default' | 'small';
             ~~~~
        The expected type comes from property 'size' which is declared here on type 'IntrinsicAttributes & { label?: ReactNode; labelPosition?: "left" | "right" | undefined; setFocus?: boolean | undefined; icon?: Element | undefined; size?: "small" | ... 1 more ... | undefined; } & Omit<...> & RefAttributes<...>'
        `